docs/usage/configuration-options.md

```diff
@@ -2108,10 +2108,11 @@ Consider this example:
 
 With the above config, every PR raised by Renovate will have the label `dependencies` while PRs containing `eslint`-related packages will instead have the label `linting`.
 
-Renovate only adds labels when it creates the PR, which means:
+Behaviour details:
 
-- If you remove labels which Renovate added, it won't re-apply them
-- If you change your config, the new/changed labels are not applied to any open PRs
+- Renovate adds labels to the PR at time of PR creation, if any have been configured
+- If any other bot or users modifies labels in a PR which Renovate created, it won't touch labels again in that open PR
+- If the labels for a PR change through configuration or other means, they are applied to any open PRs with unmodified labels (GitHub, GitLab, and Gitea only)
```

Comment:
```suggestion
- On GitHub, GitLab and Gitea: if the labels for a PR change through configuration or other means, the changes will be applied to any unmodified labels
```

It's not clear to me what is meant precisely by: `through configuration or other means`. Do you mean something like this?

> If you as user, change the labels in the Renovate config, _or_ via GitHub's, GitLab's or Gitea's UI, then those changes will be applied to any open PRs with unmodified labels?

Reply:
I only mean through configuration :)